main.js

```diff
@@ -0,0 +1,81 @@
+const app = () => {
+  const song = document.querySelector(".song");
+  const play = document.querySelector(".play");
+  const outline = document.querySelector(".moving-outline circle");
+  const video = document.querySelector(".vid-container video");
+
+  // Sounds
+  const sounds = document.querySelectorAll(".sound-picker button");
+  // Time display
+  const timeDisplay = document.querySelector(".time-display");
+  const timeSelect = document.querySelectorAll(".time-select button");
+  // Get the length of outline
+  const outlineLength = outline.getTotalLength();
+  // Duration
+  let fakeDuration = 600;
+
+  outline.style.strokeDasharray = outlineLength;
+  outline.style.strokeDashoffset = outlineLength;
+
+  //   Pick different sounds
+  sounds.forEach((sound) => {
+    sound.addEventListener("click", function () {
+      song.src = this.getAttribute("data-sound");
+      video.src = this.getAttribute("data-video");
+      checkPlaying(song);
+    });
+  });
+
+  //   Select sound
+  timeSelect.forEach((option) => {
+    option.addEventListener("click", function () {
+      fakeDuration = this.getAttribute("data-time");
+      timeDisplay.textContent = `${Math.floor(fakeDuration / 60)}:0${Math.floor(
+        fakeDuration % 60
+      )}`;
+    });
+  });
+
+  // Play sound
+  play.addEventListener("click", () => {
+    checkPlaying(song);
+  });
+
+  // Stop and play sounds
+  const checkPlaying = (song) => {
```

Comment:
Name of the method is quite clear: what the method should check ? Ideal name of the method is when you do not need to go inside it to understand what method is doing.
Something like, playOrPause() will be better.
Check article about meaningful names https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c